cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -273,6 +274,9 @@ function loadCounties() {
       // Inject each county into the DOM.
       const parseCountyData = resp.data.data;
       $.each( parseCountyData, function( i, countyData ) {
+        if ( countyData.county ) {
+          countyData.county = countyData.county.replace( ' County', '' );
```

Comment:
You could do the possibly more efficient but less clear `countyData.county.slice(0, -7)` too. But it should have a code comment that it's stripping the county in that case. Not sure it would be faster either, but I suspect it would since it wouldn't need to match the characters.